.gitignore

```diff
@@ -17,8 +17,10 @@ coverage.txt
 
 **/.DS_Store
 
-assets
-cache
+
+#added _ to exclude from gopls linting, specifically helps when working on MacOS
+_assets
+_cache 
```

Comment:
Perhaps it would be better to add the `_assets` and `_cache` rename to a separate PR, to make this one easier to review and merge? I think the changes are not necessary for the Ubuntu fix state functionality, but more of a development nice-to-have if you're using an editor which is helped by this change.

Reply:
I agree. I will remove these changes from this MR